src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
 public class WorkWithFile {
+    private static final char CSV_DELIMITER = ',';
+    private static final String FILTER_FOR_DELIMITERS = "\\W+";
+    private static final String SUPPLY_FILTER = "supply";
+    private static final String BUY_FILTER = "buy";
+    private static final int RESET_STRING_BUILDER = 0;
+    private static final int INDEX_OF_FIRST_ELEMENT = 0;
+    private static final int INDEX_OF_SECOND_ELEMENT = 1;
+    private static final int INDEX_OF_AMOUNT_ELEMENT = 1;
+    private static final int QUANTITY_OF_OPERATIONS = 2;
+    private final StringBuilder reportBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] temperedData = readFromFile(fromFileName);
+        String report = makeReport(temperedData);
+        writeToFile(report, toFileName);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        String seizedData;
+        try {
+            seizedData = Files.readString(Path.of(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read from file..." + e);
```

Comment:
please specify the file name here

Reply:
You mean create uniqe name instead of reuse "fromFileName"?